test/functional/test-custom-element.js

```diff
@@ -230,6 +242,24 @@ describe('CustomElement', () => {
     expect(testElementBuildCallback.callCount).to.equal(1);
   });
 
+  it('Element - build NOT allowed when in template', () => {
+    const element = new ElementClass();
+    expect(element.classList.contains('-amp-element')).to.equal(true);
+    expect(element.isBuilt()).to.equal(false);
+    expect(element.classList.contains('-amp-notbuilt')).to.equal(true);
+    expect(element.classList.contains('amp-notbuilt')).to.equal(true);
```

Comment:
Will keep for now. These are older tests just refactored. I think it'd be nice to do a big sweep through all these classList.contains checks.